src/main/java/io/candydoc/ddd/tactical_pattern/Repository.java

```diff
@@ -0,0 +1,24 @@
+package io.candydoc.ddd.tactical_pattern;
+
+import io.candydoc.ddd.model.DDDPattern;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class Repository extends TacticalModel {
```

Comment:
DomainRepository not to be confused with the Repository annotation of Spring (for people using Spring and Candy-Doc on their projects)